test/unit/convertV2.test.js

```diff
@@ -142,8 +141,9 @@ describe('The convert Function', function() {
       expect(conversionResult.output[0].type).to.equal('collection');
       expect(conversionResult.output[0].data).to.have.property('info');
       expect(conversionResult.output[0].data).to.have.property('item');
-      body = conversionResult.output[0].data.item[1].response[0].body;
-      expect(body).to.contain('<Error: Too many levels of nesting to fake this schema>');
+      // body = conversionResult.output[0].data.item[1].response[0].body;
+
+      // expect(body).to.contain('<Error: Too many levels of nesting to fake this schema>');
```

Comment:
@barshan23 this is getting resolved completely.
i think as soon as i fixed the bug for `seenRef` this started getting resolved completely.